app/js/services.js

```diff
@@ -0,0 +1,57 @@
+floripaApp.factory('searchService', function () {
+    return {};
+});
+
+floripaApp.factory('FindRoutesByStopName', ['$resource', '$http',
```

Comment:
It was a solution to add the Authorization header, I tried to include it inside of "save" but it didn't work.